apps/commudle-admin/src/app/feature-modules/public-community/components/events/events.component.scss

```diff
@@ -1,127 +1,117 @@
 nb-card {
-  @apply com-mt-8 com-mx-0 com-mb-4;
+  @apply com-mx-0 com-my-4;
 
   nb-card-body {
-    @apply com-p-0;
-  }
-}
-
-.events {
-  @apply com-bg-white com-rounded-xl com-gap-8 com-p-8;
-
-  a {
-    text-decoration: none;
-    @apply com-relative;
-  }
-
-  nb-card {
-    @apply com-border com-border-solid com-border-Silver-Sand com-m-0 com-overflow-hidden;
-  }
-
-  .event-clickable {
-    @apply com-w-full;
-
-    .event {
-      @apply com-w-318px hover:com-shadow-Card;
-      @apply md:com-w-full;
+    @apply com-p-0 com-overflow-hidden;
 
-      nb-card-body {
-        @apply com-flex-col;
-        @apply md:com-flex-row;
+    .events {
+      @apply com-bg-white com-rounded-xl com-flex com-flex-wrap;
 
-        .header-image {
-          @apply com-m-0 com-w-full com-overflow-hidden com-min-w-[220px] com-h-32;
-          @apply md:com-mr-6 md:com-w-220px;
-
-          img {
-            @apply com-w-full com-h-full com-object-cover com-object-left-top;
-          }
+      .heading {
+        @apply com-font-bold com-text-[28px] com-mt-9 com-mb-6 com-text-center;
+        @apply md:com-font-medium md:com-text-[32px] md:com-text-start  md:com-my-8;
+        @apply com-leading-4 com-text-black;
```

Comment:
don't use multiple line for this,
try to use in single line

Reply:
Done